assets/dbinit.sh

```diff
@@ -13,9 +13,11 @@ export CANVAS_LMS_ACCOUNT_NAME="Canvas Docker"
 export CANVAS_LMS_STATS_COLLECTION="opt_out"
 
 cd /opt/canvas/canvas-lms \
-    && $GEM_HOME/bin/bundle exec rake db:initial_setup
+    && /opt/canvas/.gem/ruby/2.7.0/bin/bundle _2.2.19_ exec rake db:initial_setup
 
-psql -U canvas -d canvas_development -c "INSERT INTO developer_keys (api_key, email, name, redirect_uri) VALUES ('test_developer_key', 'canvas@example.edu', 'Canvas Docker', 'http://localhost:8000');"
+psql -U canvas -d canvas_development -c "INSERT INTO developer_keys (api_key, email, name, redirect_uri, root_account_id, access_token_count) VALUES ('test_developer_key', 'canvas@example.edu', 'Canvas Docker', 'http://localhost:8000', 1, 1);"
```

Comment:
Ran into a build error during dbinit due to missing timestamps, but worked like a charm after the following change: 

```suggestion
psql -U canvas -d canvas_development -c "INSERT INTO developer_keys (api_key, email, name, redirect_uri, root_account_id, access_token_count, created_at, updated_at) VALUES ('test_developer_key', 'canvas@example.edu', 'Canvas Docker', 'http://localhost:8000', 1, 1, NOW(), NOW());"
```